semcore/feedback-form/src/component/feedback-item/FeedbackItem.tsx

```diff
@@ -50,9 +50,11 @@ export function FeedbackItem({
             'aria-invalid': errorState ? true : false,
             'aria-errormessage': popperId,
             'aria-describedby': errorState && meta.active ? popperId : undefined,
+            'aria-haspopup': errorState && meta.active ? true : undefined,
```

Comment:
not sure about it. 
`aria-haspopup` will come from Tooltip.

Reply:
It's actually fixes issue, with wrong element marked with aria-haspopup. It's needed here, inside FeedbackItem

Reply:
ok, let's stay